### PR TITLE
add grayscale filter effect

### LINKED ISSUE #91: [FEATURE] New fitler: grayscale
After reading issue #69, I felt it would be nice to have a grayscale filter- because grayscale images are minimalistic and is a pretty standard filter.
I haven't looked into the implementation but I guess it should be simple enough.
It would be appreciated if this issue could be assigned to me.

### PR DESCRIPTION
**Pull request by codephile1221**
Closes #91 , implemented grayscale effect.

Before filter:
skunkbooth-no-filters

After filter:
skunkbooth-grayscale-effect

Menu:
skunkbooth

### CODE REVIEW

**Review by Davidy22 — CHANGES_REQUESTED** [review]
Code works, mostly good, just one oversight to clean up

File: skunkbooth/filters/grayscale.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from PIL import Image
+from PIL.ImageOps import grayscale
+
+
+class filter:
+    """Sample filter for PIL operations"""
```
Comment (Davidy22) [inline skunkbooth/filters/grayscale.py:6]: The file you based this on was a sample filter, but for this filter you've written this should be changed to describe the filter.

Comment (codephile1221) [reply skunkbooth/filters/grayscale.py:6]: That does make more sense, haha.

**Comment by Davidy22** [comment]
Looks fine, merging.

**Pull request merged**